RootOutputerConfig.h

```diff
@@ -13,6 +13,7 @@ namespace cce::tf {
     int basketSize_=16384;
     int treeMaxVirtualSize_=-1;
     int autoFlush_=-1;
+    int cacheSize_=102400;
```

Comment:
As this object is shared across all the Root based Outputers, please modify the other ones to be able to use the `TFileCacheWrite` and set this default to
```suggestion
    int cacheSize_=0;
```